### PR TITLE
Bring kube-version and rancher-version changes to dev

### PR DESCRIPTION
**Pull request by PennyScissors**
https://github.com/rancher/charts/pull/1964

### CODE REVIEW

File: release.yaml [hunk 1]
```diff
@@ -1,32 +1,149 @@
 rancher-monitoring:
+- 100.0.0+up16.6.0
```
Comment (aiyengar2) [inline release.yaml:2]: we need to remove all the extra entries from the release.yaml here

File: packages/rancher-wins-upgrader/package.yaml [hunk 2]
```diff
@@ -2,3 +2,4 @@ url: https://github.com/rancher/wins.git
 subdirectory: charts/rancher-wins-upgrader
 commit: 4d46d8dd81cd741a51313923d459f28beea56800
 version: 100.0.1
+doNotRelease: true
```
Comment (aiyengar2) [inline packages/rancher-wins-upgrader/package.yaml:5]: can we remove all the doNotRelease tags in favor of just making the modification to the latest package checked in?

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

File: release.yaml [hunk 3]
```diff
@@ -146,4 +146,5 @@ harvester-cloud-provider:
 harvester-csi-driver:
 - 100.0.0+up0.1.9
 - 100.0.1+up0.1.9
-- 100.0.2+up0.1.11
\ No newline at end of file
+- 100.0.2+up0.1.11
+- 100.0.2+up0.1.14
```
Comment (PennyScissors) [inline release.yaml:34]: Found this issue, will fix in a separate PR once this is merged.

File: release.yaml (hunk 1 above)
Comment (PennyScissors) [reply release.yaml]: updated

File: packages/rancher-wins-upgrader/package.yaml (hunk 2 above)
Comment (PennyScissors) [reply packages/rancher-wins-upgrader/package.yaml]: updated

**Pull request merged**